apps/src/lib/util/javascriptMode.js

```diff
@@ -42,7 +42,8 @@ export function apiValidateType(
   varName,
   varValue,
   expectedType,
-  opt
+  opt,
```

Comment:
This isn't part of your change, but do you know what the difference between `opts` and `opt` is?

Reply:
`opts` is an object and `opt` is a boolean and the value for `true` meaning 'Optional`.